src/internal.c

```diff
@@ -1707,7 +1708,7 @@ static int GetNameList(byte* idList, word32* idListSz,
 }
 
 
-static const byte  cannedEncAlgo[] = {ID_AES128_GCM, ID_AES128_CBC};
+static const byte  cannedEncAlgo[] = {ID_AES128_GCM, ID_AES128_CTR, ID_AES128_CBC};
```

Comment:
When you add items to the cannedXxxAlgo lists, you need to add the corresponding names to the cannedXxxAlgoNames lists. (In the future, this is something that needs to be configurable.)

Reply:
added. Still need to be configurable.